### ISSUE TITLE
Extracts - Phytoplasma as an attribute

### ISSUE THREAD

**Issue by mmkohler** [SpeciesFileGroup/taxonworks#3976]
### Feature or enhancement

When I am in New Extracts, I want to be able to associate the extract with a  possible Phytoplasma. I want a box that allows me to click on it to say yes or no a phytoplasma was associated with this extract. Also will need to be able to input an exact value. Perhaps two seperate enhancements. 

### Location

New Extracts

**Comment by debpaul** [comment]
@mmkohler when you say you want to input an exact value (in addition to a boolean y/n flag) what would be examples of some values you would want to store?

**Comment by mmkohler** [comment]
Valeria and her group would like to put a numerical value associated with the Phytoplasma occurance

**Comment by mjy** [comment]
Yes, we need to understand the semantics of the "value" to best recommend a modelling approach.

**Comment by proceps** [comment]
my guess they may want to create a new key word for the data attribute and add this to the extract form. But I could be wrong...

**Comment by mmkohler** [comment]
The group would like to input decimal number associated with the phytoplasma. It might be easier to do an Associate category, similar to the Biological Associations in the comprehensive.
